app/pods/components/otp-enroll/component.js

```diff
@@ -42,7 +45,10 @@ export default class otpEnrollComponent extends Component {
       window.location.reload()
     }).catch(err => {
       console.error(err)
-      this.set('errorString', 'Incorrect OTP');
+      const errorMsg = JSON.parse(err);
+      if (typeof errorMsg == Object)
```

Comment:
keep your code DRY, extract this to a util/method maybe?

Reply:
done